sitemap-scraper/src/internals/tools.ts

```diff
@@ -0,0 +1,126 @@
+import type { ProxyConfiguration } from 'crawlee';
```

Comment:
this file was fully copied from WCC, but I'd like to move these helpers into a shared lib